Header header

string data
uint8 total_number